ReactWindows/ReactNative.Shared/ReactInstanceManager.cs

```diff
@@ -400,17 +448,17 @@ public IReadOnlyList<IViewManager> CreateAllViewManagers(ReactContext reactConte
             }
         }
 
-        private Task<ReactContext> CreateReactContextCoreAsync(CancellationToken token)
+        private async Task<ReactContext> CreateReactContextCoreAsync(CancellationToken token)
```

Comment:
>async [](start = 16, length = 5)

why does this have to be converted to async/await?

Reply:
You're right, perhaps an automatism. I fixed this and the next method.

---
In reply to: [166568092](https://github.com/Microsoft/react-native-windows/pull/1615#discussion_r166568092) [](ancestors = 166568092)